ex7-structCond.js

```diff
@@ -0,0 +1,21 @@
+// - Exercice 7
+// - Ecrire un algorithme qui prend 3 nombres en entrée, stockés
+// dans 3 variables
+// - Vérifier si les nombres sont donnés dans l’ordre croissant
+// - Amélioration : remettez-les dans l’ordre
+
+let prompt = require("prompt-sync")();
+let a = parseFloat(prompt("Saisir le premier nombre :"));
+let b = parseFloat(prompt("Saisir le deuxième nombre :"));
+let c = parseFloat(prompt("Saisir le troisième nombre :"));
```

Comment:
juste il faut eviter d'utiliser les nom des variable comme ça 
il faut toujours préciser le role du variable dans le nom
par exemple 
```suggestion
let firstNumber = parseFloat(prompt("Saisir le premier nombre :"));
let secondNumber = parseFloat(prompt("Saisir le deuxième nombre :"));
let thirdNumber = parseFloat(prompt("Saisir le troisième nombre :"));
```

Reply:
bien compris